pkg/k8s/apis/cell.go

```diff
@@ -61,13 +70,28 @@ func createCRDs(p params) {
 				return nil
 			}
 
-			if err := RegisterCRDs(p.Clientset); err != nil {
-				return fmt.Errorf("unable to create CRDs: %w", err)
+			for _, f := range p.RegisterCRDsFuncs {
+				if err := f(p.Clientset); err != nil {
+					return fmt.Errorf("unable to create CRDs: %w", err)
+				}
 			}
+
 			return nil
 		},
 		OnStop: func(_ hive.HookContext) error {
```

Comment:
No need to provide an implementation for the stop hook. `nil` OnStop will be skipped.